programs/dynamic-bonding-curve/src/instructions/migration/dynamic_amm_v2/migrate_damm_v2_initialize_pool.rs

```diff
@@ -679,3 +950,36 @@ fn get_liquidity_for_adding_liquidity(
             .map_err(|_| PoolError::TypeCastFailed)?)
     }
 }
+
+fn get_damm_v2_vesting_parameters(
+    vested_liquidity: u128,
+    lp_vesting_info: &LpVestingInfo,
+) -> Result<damm_v2::types::VestingParameters> {
+    let clock = Clock::get()?;
+    let current_timestamp = clock.unix_timestamp as u64;
```

Comment:
we could pass current_timestamp from outside to avoid double `    let clock = Clock::get()?;`